spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesDiscoveryClientAutoConfiguration.java

```diff
@@ -1,132 +0,0 @@
-/*
```

Comment:
this class was refactored and simplified into `KubernetesInformerDiscoveryClientAutoConfiguration `